### PR TITLE
Add mapbiomas alerts code

### PR DESCRIPTION
**Pull request by jjcastro**

### CODE REVIEW

File: fetch_mapbiomas_alerts.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import geopy.distance
+import requests
+import json
+import ee
+
+def fetch_alerts(lat: float, lon: float):
```
Comment (benwulfe) [inline fetch_mapbiomas_alerts.py:6]: doccomment

**Review by benwulfe — CHANGES_REQUESTED** [review]
Please add a test that double checks that our expected contract with mapbiomas remains intact. 
Also please ensure its resilient to failure -- especially important given this is an external entity.

**Comment by jjcastro** [comment]
Done, created a test to test the contracts and added documentation

**Review by benwulfe — APPROVED** [review]
This is excellent. Fronza was looking for how to do this, so if you can come to next tuesday session, we can review this and the frontend code with him.

**Pull request merged**